csgo/addons/sourcemod/scripting/ckSurf/surfzones.sp

```diff
@@ -418,7 +422,8 @@ public Action BeamBoxAll(Handle timer, any data)
 								buffer_a[x] = g_mapZones[i][PointA][x];
 								buffer_b[x] = g_mapZones[i][PointB][x];
 							}
-							TE_SendBeamBoxToClient(p, buffer_a, buffer_b, g_BeamSprite, g_HaloSprite, 0, 30, GetConVarFloat(g_hChecker), 5.0, 5.0, 2, 1.0, tzColor, 0, 0, i);
+							// stock void TE_SendBeamBoxToClient(int client, float uppercorner[3], float bottomcorner[3], int ModelIndex, int HaloIndex, int StartFrame, int FrameRate, float Life, float Width, float EndWidth, int FadeLength, float Amplitude, const int Color[4], int Speed, int type, int zoneid = -1)
+							TE_SendBeamBoxToClient(p, buffer_a, buffer_b, g_BeamSprite, g_HaloSprite, 0, 30, GetConVarFloat(g_hChecker), 1.0, 1.0, 0, 0.0, tzColor, 0, 0, i);
```

Comment:
Is this a wanted change?
can you confirm?

Reply:
It changes the zone width and the amplitude which causes the "shaking" effect or whatever you want to call it, I very much doubt it affects FPS but I find it quite annoying, also allows people to make their zones more accurate, up to you though.